.travis.yml

```diff
@@ -19,19 +23,33 @@ env:
     - ZEPHYR_SDK_DOWNLOAD_FOLDER=https://nexus.zephyrproject.org/content/repositories/releases/org/zephyrproject/zephyr-sdk/$ZEPHYR_SDK_VERSION-i686
     - ZEPHYR_SDK_SETUP_BINARY=zephyr-sdk-$ZEPHYR_SDK_VERSION-i686-setup.run
     - ZEPHYR_SDK_DOWNLOAD_URL=$ZEPHYR_SDK_DOWNLOAD_FOLDER/$ZEPHYR_SDK_SETUP_BINARY
-  matrix:
-    - TARGET="zephyr"
-    - TARGET="extra"
-    - TARGET="linux"
-    - TARGET="ashell"
+matrix:
+  include:
+    - os: linux
+      env: TARGET="zephyr"
+    - os: linux
+      env: TARGET="extra"
+    - os: linux
+      env: TARGET="linux"
+    - os: linux
+      env: TARGET="ashell"
+    - os: osx
+      osx_image: xcode8.3
+      env: TARGET="linux"
```

Comment:
tthe os: osx is in the last one already, so the top 4 is os: linux, the 5th one is os: osx, and if you look at the build status page, the osx one has a apple logo instead of penguin